src/PrestaShopBundle/Controller/Admin/ErrorController.php

```diff
@@ -34,7 +34,7 @@
 /**
  * Manages Error pages (e.g. 500)
  */
-class ErrorController extends FrameworkBundleAdminController
+class ErrorController extends PrestaShopAdminController
```

Comment:
This should be detailed in the How to test to know how (probably a developer) will have to test this PR